public/theme/boltcms-2021/partials/_master.twig

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang={{ htmllang() }}>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Work+Sans:wght@100..900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="{{ asset('css/app.css', 'theme') }}">
+    <link rel="stylesheet" href="https://unpkg.com/transition-style">
+    {# <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/tiny-slider/2.9.3/tiny-slider.css"> #}
```

Comment:
let's delete the commented out code, if it's not used. :-)

Reply:
Check